tests/conftest.py

```diff
@@ -275,6 +275,16 @@ def min_mender_client_version(request):
         pytest.skip("Test requires Mender client %s or newer" % min_required_version)
 
 
+@pytest.fixture(autouse=True)
+def mender_client_version():
+    mender_client_version = (
+        subprocess.check_output([RELEASE_TOOL, "--version-of", "mender"])
+        .decode()
+        .strip()
+    )
+    return mender_client_version
```

Comment:
Suggestion: 

How about just returning the statement straight up, we already know from the function name what's going on?

Reply:
done